### DISCUSSION TITLE
Blueprint - Texture out not recognized as type texture

### DISCUSSION THREAD

**Discussion by franckaubry** [cables-gl/cables_docs#322] [! Welcome - readme first!]
Creating an texture out port in a SubPatch works fine, but when "converted" to a Blueprint, the texture out port is not longer recognized as an texture port
`Object in port Texture In is not of type texture`

The VizTexture does draw the texture in it's preview but nothing is shown in the render window. 

![Screenshot by Dropbox Capture](https://user-images.githubusercontent.com/97125854/185604788-16c81106-bc37-42c6-a051-d0c8813b9f3b.png)

**Comment by franckaubry** [comment]
Patch secret URL: https://dev.cables.gl/p/62fb5b0ea1293e86e06c04d4?s=nwjkJTIMuen6kwou

**Comment by steam0r** [comment]
you dont have basicmaterial/rectangle triggered in that patch, this is why it's not showing anything. it seems to be fine apart from showing that error message.

the "not of type texture" actually is a bug that i have to fix, thanks for reporting!

**Reply by franckaubry** [reply]
Ops! A bit quick with that one yes! I disconnected them for further bug testing and apparently forgot to connect again. Sorry.

**Reply by steam0r** [reply]
this is now fixed on dev. thanks!
